sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/Optimizer.scala

```diff
@@ -68,6 +68,7 @@ abstract class Optimizer extends RuleExecutor[LogicalPlan] {
       PushPredicateThroughAggregate,
       ColumnPruning,
       // Operator combine
+      RepartitionCollapsing,
```

Comment:
Sure; I prefer the "active voice" naming convention that's used by the other operators. While we're at it, we should probably also rename `ColumnPruning` to `PruneColumns`.

I guess this renaming is going to break third-party code which relies on these rule names, but that's fine since we're not doing it in a minor release.